lib/main_common.pm

```diff
@@ -1460,9 +1460,9 @@ sub load_extra_tests {
     return if get_var("INSTALLONLY") || get_var("DUALBOOT") || get_var("RESCUECD");
 
     # setup $serialdev permission and so on
-    loadtest "console/consoletest_setup";
+    loadtest "console/consoletest_setup" unless get_var("BOOT_HDD_IMAGE");
     loadtest 'console/integration_services' if is_hyperv;
-    loadtest "console/hostname";
+    loadtest "console/hostname" unless get_var("BOOT_HDD_IMAGE");
```

Comment:
Thanks, I got to know that.  I noticed that the "x11/x11_setup" is similar with "console/setup_serialdev" in #4909 and since #4909 has not been merged, so I used x11_setup as substitution.  But I will change "x11/x11_setup" to "console/setup_serialdev" once it's been merged if necessary.